src/module_loader.rs

```diff
@@ -145,51 +207,29 @@ impl ModuleLoader for RustyLoader {
     ) -> deno_core::ModuleLoadResponse {

```

Comment:
Shouldn't those additional parameters be available to the trait

Reply:
I initially tried that and hit some friction. Will look into it more.